Update UDP client connection syntax to use udpin://

KarthiAru opened the pull request:
Changed all instances of the deprecated UDP connection syntax udp://:PORT to the recommended udpin://:PORT following the deprecation warning: Connection using udp:// is deprecated, please use udpin:// or udpout:// (cli_arg.cpp:28)

KarthiAru committed:
Update UDP client connection syntax to use udpin://

Changed all instances of the deprecated UDP connection syntax udp://:PORT to the recommended udpin://:PORT following the deprecation warning:
Connection using udp:// is deprecated, please use udpin:// or udpout:// (cli_arg.cpp:28)

julianoes reviewed (changes requested):
And it should be `udpin://0.0.0.0:14540` with the new syntax.

Note that I won't merge this just yet until the release is out with the changes.

KarthiAru committed:
Change UDP Binding Address

Updated the binding address for the UDP connection from udpin://:PORT to udpin://0.0.0.0:PORT

KarthiAru commented:
Done. Updated to include 0.0.0.0

julianoes commented:
Sorry, I missed to merge this before moving the docs over to https://github.com/mavlink/MAVSDK/tree/main/docs. I have since done a pass, and hopefully fixed most of these.
